lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -140,6 +141,7 @@ def friendly_inspect
 Username: #{username}
 Database: #{database}
 FRIENDLY
+      output << "Port:     #{port}\n" if port
```

Comment:
aah. I was thinking we would only display it if we wanted to change something.

same with writing to the yaml file. only write it if it is non default.

leaving it in sounds good